src/reducers/mode.js

```diff
@@ -0,0 +1,15 @@
+export default function(state={}, action) {
```

Comment:
Kan forøvrig nevne at `state` her ikke er hele state-treet, men kun `true/false`-verdiene til mode